manifest/net.rampa3/Resonite3DDashOnScreen/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "3DDashOnScreenResonite",
+	"id": "net.rampa3.3DDashOnScreenResonite",
+	"description": "Resonite Mod Loader mod, that replaces 2D overlay dash in Screen mode with the regular VR one",
+	"category": "Dashboard",
+	"sourceLocation": "https://github.com/rampa3/3DDashOnScreenResonite",
+	"versions": {
+		"3.6.0": {
+			"releaseUrl": "https://github.com/rampa3/3DDashOnScreenResonite/releases/tag/3.6.0.0",
+			"artifacts": [{
```

Comment:
Nonblocking: Looks like `[assembly: AssemblyInformationalVersion("1.0.0")]` is set (or default). If using only csproj adding `<Version>3.6.0</Version>` should work to set all of these

Reply:
Will fix in source. "AssemblyInformationalVersion" is not set in the csproj, so if it shows 1.0.0, it must be defaulting.

Reply:
Should I redo the current release too?

Reply:
Not required but if you want to update it, I'll relook at it instead of merging this one in.

Reply:
OK, gimme a moment - have to fix the code, rebuild, force delete the tag and republish it.